### ISSUE TITLE
Require: Gossip/real-time exchange protocol

### ISSUE THREAD

**Issue by Kiddinglife** [nanomsg/nng#1492]
some stable and useful RFCs from zeromq are NOT implemented in NNG. 
https://rfc.zeromq.org/

**Comment by gdamore** [comment]
Of course, ZMQ is not NNG and has a different development history.  However, I'll keep this ticket open as an RFE to add a similar protocol in the future.
